### PR TITLE
[ISSUE #6392]GetSyncStateSet prints replica's alive status

### PR DESCRIPTION
**Pull request by GenerousMan**
**Make sure set the target branch to `develop`**

## What is the purpose of the change

GetSyncStateSet prints replica's alive status

fix ISSUE #6392 

## Brief changelog

print whether the broker is alive or not.

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Comment by GenerousMan** [comment]
@RongtongJin

File: remoting/src/main/java/org/apache/rocketmq/remoting/protocol/body/BrokerReplicasInfo.java [hunk 1]
```diff
@@ -115,29 +114,32 @@ public Long getMasterBrokerId() {
         }
 
         public boolean isExistInSync(String brokerName, Long brokerId, String brokerAddress) {
-            return this.getInSyncReplicas().contains(new ReplicaIdentity(brokerName, brokerId, brokerAddress));
+            return this.getInSyncReplicas().contains(new ReplicaIdentity(brokerName, brokerId, brokerAddress, true))
+                    || this.getInSyncReplicas().contains(new ReplicaIdentity(brokerName, brokerId, brokerAddress, false)) ;
```
Comment (TheR1sing3un) [inline remoting/src/main/java/org/apache/rocketmq/remoting/protocol/body/BrokerReplicasInfo.java:118]: The `ReplicaIndentity`'s field `isAlive` will not participate in the method `List::contains`. How about adding a constucter method like `public ReplicaIndentity(String brokerName, Long brokerId, String brokerAdress)` and the filed `isAlive` with the default value `false` ~

Comment (GenerousMan) [reply remoting/src/main/java/org/apache/rocketmq/remoting/protocol/body/BrokerReplicasInfo.java:118]: You are right. Thanks a lot~

**Review by TheR1sing3un — APPROVED** [review]
LGTM~

**Review by RongtongJin — APPROVED** [review]

File: remoting/src/main/java/org/apache/rocketmq/remoting/protocol/body/BrokerReplicasInfo.java [hunk 2]
```diff
@@ -164,12 +163,21 @@ public void setBrokerId(Long brokerId) {
             this.brokerId = brokerId;
         }
 
+        public Boolean getIsAlive() {
+            return this.isAlive;
+        }
+
+        public void setIsAlive(Boolean isAlive) {
+            this.isAlive = isAlive;
+        }
```
Comment (RongtongJin) [inline remoting/src/main/java/org/apache/rocketmq/remoting/protocol/body/BrokerReplicasInfo.java:172]: How about renaming isAlive and setAlive ？

**Review by RongtongJin — APPROVED** [review]

**Pull request merged**